### PR TITLE
Add unit test for InterpolatedDocumentFieldLookupNode.

### PR DESCRIPTION
**Pull request by toregge**
@geirst : please review

### CODE REVIEW

File: searchlib/src/tests/expression/interpolated_document_field_lookup_node/interpolated_document_field_lookup_node_test.cpp [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+{
+protected:
+    InterpolatedDocumentFieldLookupNodeTest();
+    ~InterpolatedDocumentFieldLookupNodeTest() override;
+};
+
+InterpolatedDocumentFieldLookupNodeTest::InterpolatedDocumentFieldLookupNodeTest() = default;
+InterpolatedDocumentFieldLookupNodeTest::~InterpolatedDocumentFieldLookupNodeTest() = default;
+
+TEST_F(InterpolatedDocumentFieldLookupNodeTest, hello_world)
```
Comment (geirst) [inline searchlib/src/tests/expression/interpolated_document_field_lookup_node/interpolated_document_field_lookup_node_test.cpp:88]: Please renaming test function to reflect what is being tested.

**Review by geirst — DISMISSED** [review]
👍

File: searchlib/src/tests/expression/interpolated_document_field_lookup_node/interpolated_document_field_lookup_node_test.cpp (hunk 1 above)
Comment (toregge) [reply searchlib/src/tests/expression/interpolated_document_field_lookup_node/interpolated_document_field_lookup_node_test.cpp]: Renamed.

**Review by geirst — APPROVED** [review]
👍

**Pull request merged**